src/run-on-arch.sh

```diff
@@ -3,17 +3,23 @@
 set -euo pipefail
 
 # Args
-DOCKERFILE=$1
-CONTAINER_NAME=$2
+ARCH=$1
+DOCKERFILE=$2
+CONTAINER_NAME=$3
 # Remainder of args get passed to docker
-declare -a DOCKER_RUN_ARGS=${@:3:${#@}}
+declare -a DOCKER_RUN_ARGS=${@:4:${#@}}
 
 # Defaults
 ACTION_DIR="$(cd "$(dirname "$0")"/.. >/dev/null 2>&1 ; pwd -P)"
 LOWERCASE_REPOSITORY=$(printf "%s" "$GITHUB_REPOSITORY" | tr '[:upper:]' '[:lower:]')
 PACKAGE_REGISTRY="ghcr.io/${LOWERCASE_REPOSITORY}/${CONTAINER_NAME}"
 DEBIAN_FRONTEND=noninteractive
 
+# Sanitize inputs
+if [[ "$ARCH" == "none" ]]; then
+  unset ARCH
```

Comment:
```suggestion
  export ARCH=amd64
```

Reply:
I was thinking if we should allow an interface to not pass `--platform`, that was the reason for `unset` approach